docs/integrations/aws.md

```diff
@@ -0,0 +1,388 @@
+---
+title: AWS
+last_update:
+  date: 2025-09-28
+---
+
+## Overview
+
+Statsig now offers support to use statsig configs on AWS edge services, providing low latency for gate and experiment evaluations in your AWS project.
+
+This integration uses the bring your own CDN model from the statsig edge client. This guide will highlight the most optimal setup for using statsig with AWS to leverage performance on the edge.
+
+## Architecture
+
+This integration uses the following AWS services:
+
+- **S3**: Centralized configuration storage to store your statsig configs
+- **Lambda@Edge**: Executes your project code which uses the statsig edge SDK at CloudFront edge locations
+- **CloudFront**: Global CDN that triggers Lambda@Edge functions and caches responses
+- **IAM**: Manages permissions for Lambda@Edge functions to access S3
+- **CloudWatch**: Provides logging and monitoring for your edge functions
+
+## Prerequisites
+
+**Statsig**:
+
+- Statsig account
+- Configured statsig project
+- A 50/50 feature gate for testing. This guide refers to this gate as "test_gate"
+- Client API Key (**Settings** -> **Keys & Environments**)
+
+**AWS**:
+
+- AWS Account with administrative access
+
+## S3 Storage Setup
+
+The S3 bucket will be leveraged to store your statsig Config specifications. Since Amazon does not provide a traditional KV store for lambda@edge, we will treat the S3 storage as our KV store.
```

Comment:
dont use passive voice. "will be leveraged"

"For edge integrations, the statsig SDK needs access to storage located as close to the lambda as possible.  On aws, we recommend storing your statsig project definition in an s3 bucket" or something like that.

Also, can you make this generic?  Im pretty sure you can sub gcp function / gcp data store in for lambda/s3